mod_reforged/config/z_nested_tooltips.nut

```diff
@@ -21,6 +21,7 @@
 			Turn = ::MSU.Class.BasicTooltip("Turn", ::Reforged.Mod.Tooltips.parseString("Combat in battle brothers is turn-based. Each combat consists of a series of [rounds|Concept.Round]. During a round, characters act in turns. A character\'s position in the turn order is determined by the character\'s [Initiative|Concept.Initiative] relative to other characters.\n\n[Effects|Concept.StatusEffect] that last a certain number of turns last until the character has started or ended their turn that many times, depending on whether the [effect|Concept.StatusEffect] expires at the start or end of the turn respectively.")),
 			Round = ::MSU.Class.BasicTooltip("Round", ::Reforged.Mod.Tooltips.parseString("Combat in battle brothers is turn-based. Each combat consists of a series of rounds. During a round, characters act in [turns|Concept.Turn]. A round ends when all characters have ended their [turns|Concept.Turn].")),
 			ZoneOfControl = ::MSU.Class.BasicTooltip("Zone of Control", ::Reforged.Mod.Tooltips.parseString("Most melee characters exert Zone of Control on their surrounding tiles. Trying to move out of enemy Zone of Control will trigger one free attack from each controlling enemy until the first attack hit or every attack missed. A hit will cancel the movement.")),
+			BagSlots = ::MSU.Class.BasicTooltip("Bag Slots", ::Reforged.Mod.Tooltips.parseString("Bagslots can be used to store additional weapons or utility items for use during combat. Every character has 2 Bagslots by default and can have at most 4 Bagslots.")),
```

Comment:
- `Bag slots` with a space.
- The last sentence can end as `and can have a maximum of 4.`
- Both the default and the max number of bag slots should be dynamically read from `item_container.m.UnlockedBagSlots` and `::Const.ItemSlotSpaces` respectively. For the value from `item_container` you can instantiate an `item_container` locally above in this file and store the result of `.m.UnlockedBagSlots` in a local variable.